Modify bucket popfsenselicies template (#6769)
wazuh/wazuh-documentation: source/_templates/cloud/amazon/bucket_policies.rst

Reviewer: We could  highlight each `<bucket-name>` line where the user needs to replace with the bucket name

```
.. code-block:: json
   :emphasize-lines: 12,13

```

and

```
.. code-block:: json
   :emphasize-lines: 13,14

   {
```

![imagen](https://github.com/wazuh/wazuh-documentation/assets/47069802/3c0950e8-5533-47a0-ac83-c8d5b87178bc)

		    "s3:DeleteObject"
		],
		"Resource": [
		    "arn:aws:s3:::<bucket-name>/*",
		    "arn:aws:s3:::<bucket-name>"
		]
	    }
	]
    }

> .. note:: ``<bucket-name>`` is a placeholder, replace it with the name of the bucket where you want to fetch the logs from.

Reviewer: ```suggestion
.. note:: ``<bucket-name>`` is a placeholder. Replace it with the actual name of the bucket from which you want to retrieve logs.
```
